pages/base_test.py

```diff
@@ -88,19 +87,15 @@ def delete_run(self, mozwebqa, run, delete_version=False):
         if delete_version:
             self.delete_version(mozwebqa, version=run['version'])
 
-    def create_suite(self, mozwebqa, product, use_API, status='active', case_list=[], **kwargs):
-        if use_API:
-            credentials = mozwebqa.credentials['default']
+    def create_suite(self, mozwebqa, product, api=None, status='active', case_list=[], **kwargs):
+        if api is not None:
```

Comment:
Could this be just `if api`? `api` being an empty string would not be a valid value for `api`.